client/modules/Rewards/Rewards.jsx

```diff
@@ -33,24 +34,11 @@ function Rewards() {
       />
 
       <section className={styles.content}>
-        <p> ipsum dolor sit amet, consectetur adipiscing elit. Aliquam quis metus porta erat tempor dignissim.
-        Etiam finibus, nibh at dictum commodo, diam mi rutrum nisi, et fringilla neque odio ut turpis. Vestibulum ante ipsum
-        primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nunc at vehicula turpis, non fringilla mi. Ut consequat
-        sollicitudin purus. Vestibulum sodales nisi odio, ut pretium neque tristique cursus. Suspendisse eget magna ante. Cras quis
-        vestibulum ipsum. Nulla placerat eros scelerisque, volutpat quam sed, elementum lacus. Sed varius est magna,
-        eu hendrerit urna consequat ut. Integer at ex id tellus suscipit ultrices.</p>
-        <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam quis metus porta erat tempor dignissim.
-        Etiam finibus, nibh at dictum commodo, diam mi rutrum nisi, et fringilla neque odio ut turpis. Vestibulum ante
-        ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nunc at vehicula turpis, non fringilla mi.
-        Ut consequat sollicitudin purus. Vestibulum sodales nisi odio, ut pretium neque tristique cursus. Suspendisse ege
-        magna ante. Cras quis vestibulum ipsum. Nulla placerat eros scelerisque, volutpat quam sed, elementum lacus. Sed
-        varius est magna, eu hendrerit urna consequat ut. Integer at ex id tellus suscipit ultrices.</p>
-        <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam quis metus porta erat tempor dignissim.
-        Etiam finibus, nibh at dictum commodo, diam mi rutrum nisi, et fringilla neque odio ut turpis. Vestibulum ante ipsum
-        rimis in faucibus orci luctus et ultrices posuere cubilia Curae; Nunc at vehicula turpis, non fringilla mi. Ut consequ
-        t sollicitudin purus. Vestibulum sodales nisi odio, ut pretium neque tristique cursus. Suspendisse eget magna ante. Cras
-        quis vestibulum ipsum. Nulla placerat eros scelerisque, volutpat quam sed, elementum lacus. Sed varius est magna, eu hend
-        rerit urna consequat ut. Integer at ex id tellus suscipit ultrices.</p>
+        <div className={styles.titleText}>
```

Comment:
You're going to need to import this from another SCSS file that has `.title-text`. This style class technically doesn't exist right now in `Rewards.scss`.